logging/template-logging.md

```diff
@@ -0,0 +1,70 @@
+---
+title: template for logging
+author: The Good Docs Project 
+uri-repo: https://github.com/projectname
+company: REPLACEME
+product: <!-- Set the product parameter to add the product name here. -->
```

Comment:
Few thoughts in this `YAML` block and even things I've learned..

1. Comments in YAML are only valid if you use `#` -- [Yaml Comments](https://stackoverflow.com/questions/2276572/how-do-you-do-block-comments-in-yaml)
2. Setting place holders / empty values shouldn't use `<!-- text -->` that could have issues being potable.  Better to default to an empty string or `null` in these case see [stackoverflow](https://stackoverflow.com/questions/34089496/empty-field-in-yaml#:~:text=You%20can%20use%20~%20or%20null%20.&text=If%20you%20want%20an%20empty,can%20use%20two%20single%20quotes.)

Reply:
Additionally, we haven't had a conversation yet internally on what are the `YAML` front-matter we want to support internally.  This is a discussion that we will have and if you want to participate you're welcome to join our calls.

Reply:
I will fix the comment tagging.